src/coreclr/vm/CMakeLists.txt

```diff
@@ -891,6 +895,11 @@ endif(CLR_CMAKE_TARGET_WIN32)
 set (VM_SOURCES_WKS_SPECIAL
     codeman.cpp
     ceemain.cpp
+    peimagelayout.cpp
```

Comment:
Why couldn't it stay in the common list? I have originally structured the lists so that no file needed to be duplicated in multiple lists.

Reply:
common list compiles only once - for singlefile and for regular coreclr. That is almost every file. And we reuse objects for singlefile and regular coreclr.
A few files need to know about singlefile and have parts conditionally compiled for `CORECLR_EMBEDDED`. `peimagelayout.cpp` cannot and does not need to support decompression in regular coreclr mode.